pipeline/bicleaner/bicleaner.sh

```diff
@@ -65,7 +65,7 @@ else
                # The GPU devices have failed to be found, and bicleaner AI falls back
                # to operate on the CPU very slowly. To guard against this wasting expensive
                # GPU time, always check that it can find GPUs.
-               python3 -c "import tensorflow; exit(0) if tensorflow.config.list_physical_devices('GPU') else exit(9001)"
+               python3 -c "import tensorflow; exit(0) if tensorflow.config.list_physical_devices('GPU') else exit(75)"
```

Comment:
@gregtatum it's good that you check this inside the function right before calling bicleaner because it seems GPUs can disconnect while the task is running. Please add this check also in the "else" branch where we have one GPU (I don't think we ever want to run on a CPU).

Reply:
Ah, I missed this comment before merging.